osidb/serializer.py

```diff
@@ -892,23 +937,22 @@ def create(self, validated_data):
             instance.bzsync(bz_api_key=self.get_bz_api_key())
         return instance
 
-    def update(self, instance, validated_data):
+    def update(self, instance, validated_data, *args, **kwargs):
         """
         perform the ordinary instance update
         with providing BZ API key while saving
         """
-        skip_bz_sync = validated_data.pop("skip_bz_sync", False)
-        instance = super().update(instance, validated_data)
-        if not skip_bz_sync:
-            instance.bzsync(bz_api_key=self.get_bz_api_key())
```

Comment:
Will be `bzsync` still performed after this change?

Reply:
With the correct parameters it is performed here https://github.com/RedHatProductSecurity/osidb/blob/master/apps/bbsync/mixins.py#L37 through the regular `save` method. And it is more meaningfully done as part of that single `save` call at the end of it without serializer having to care.